📖 Docs cluster controller (#5611)
kubernetes-sigs/cluster-api: docs/book/src/developer/architecture/controllers/control-plane.md

### Relationship to other Cluster API types

> The Cluster controller will set an OwnerReference on the Control Plane CR. The Control Plane controller should normally exit reconciliation until it sees the owner reference.

Reviewer: ```suggestion
The Cluster controller will set an OwnerReference on the controlPlane. The controlPlane controller should normally take no action during reconciliation until it sees the ownerReference.
```

> A Control Plane controller implementation should exit reconciliation until it sees `cluster.spec.controlPlaneEndpoint` populated.

Reviewer: ```suggestion
A controlPlane controller implementation should take no action during reconciliation until it sees `cluster.spec.controlPlaneEndpoint` populated.
```

> The Cluster controller bubbles up `status.ready` into `status.controlPlaneReady`  and `status.initialized` into a `controlPlaneInitialized` condition from the Control Plane CR.

Reviewer: ```suggestion
The Cluster controller bubbles up `status.ready` into `status.controlPlaneReady`  and `status.initialized` into a `controlPlaneInitialized` condition from the controlPlane.
```
